common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -127,107 +125,45 @@ public static String modifyMultipleMatchKeyword(String query){
-                            conceptCodeIDName.add(root.get("conceptId").in(subquery));
-                            predicates.add(
-                                    criteriaBuilder.or(
-                                            conceptCodeIDName.toArray(new Predicate[0])
-                                    )
-                            );
-                        }
-
                         predicates.add(
                                 criteriaBuilder.or(standardConceptPredicates.toArray(new Predicate[0]))
                         );
-
-                    }else if (standardConceptFilter.equals(StandardConceptFilter.NON_STANDARD_CONCEPTS)) {
-
-                        if(keyword != null) {
-                            conceptCodeIDName.add(criteriaBuilder.greaterThan(matchExp, 0.0));
-                            conceptCodeIDName.add(root.get("conceptId").in(subquery));
-                            predicates.add(
-                                    criteriaBuilder.or(
-                                            conceptCodeIDName.toArray(new Predicate[0])
-                                    )
-                            );
-                        }
-
-
+                    } else if (standardConceptFilter.equals(StandardConceptFilter.NON_STANDARD_CONCEPTS)) {
                         predicates.add(
                                 criteriaBuilder.or(
                                         criteriaBuilder.or(criteriaBuilder.isNull(root.get("standardConcept"))),
                                         criteriaBuilder.and(nonStandardConceptPredicates.toArray(new Predicate[0]))
                                 ));
 
-                    }else if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH)) {
-
+                    } else if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH)) {
                         if(keyword != null){
-                            List<Predicate> conceptNameFilter = new ArrayList<>();
-                            List<Predicate> matchFilter = new ArrayList<>();
-                            matchFilter.add(criteriaBuilder.greaterThan(matchExp, 0.0));
-                            matchFilter.add(root.get("conceptId").in(subquery));
-                            conceptNameFilter.add(criteriaBuilder.or(matchFilter.toArray(new Predicate[0])));
-                            conceptNameFilter.add(criteriaBuilder.or(standardConceptPredicates.toArray(new Predicate[0])));
-
-                            predicates.add(
-                                    criteriaBuilder.or(
-                                            criteriaBuilder.or(conceptCodeIDName.toArray(new Predicate[0])),
-                                            criteriaBuilder.and(conceptNameFilter.toArray(new Predicate[0]))
-                                    ));
+                          List<Predicate> standardOrCodeOrIdMatch = new ArrayList<>();
```

Comment:
Is the plan to eventually take this out since we have this in the the ft index now and just check the results to see if it's a code match or concept_id match?